### PR TITLE
Adding GPIO support to main branch

### PR DESCRIPTION
**Pull request by charlie353535**
To use GPIO, define USE_RPI_GPIO_PARALLEL in include/global.h
As of yet, it is **_highly_** experimental.

### CODE REVIEW

File: include/global.h [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#ifndef GLOBAL_H
```
Comment (cyclane) [inline include/global.h:1]: Nice

**Review by cyclane — APPROVED** [review]
```
#ifndef GLOBAL_H
#define GLOBAL_H
```
hmmmmmmmmmmmmm

**Comment by charlie353535** [comment]
Cool. All done.

File: include/global.h (hunk 1 above)
Comment (charlie353535) [reply include/global.h:1]: Hmm indeed,

**Pull request merged**